### ISSUE TITLE
Add a settings page that allows you to enable / disable the extension on certain urls

### ISSUE THREAD

**Issue by trosel** [eraeco/party.lol#3]
The settings could be similar to the [Redirector](https://addons.mozilla.org/en-US/firefox/addon/redirector/) extension.

You add in certain URLs with wildcards. That way I could use the extension with a specific subreddit and not a different subreddit, for example.

Additionally, it could be good to add a config for specific elements on the website that would trigger the extension and not others. For example, maybe I want the extension to trigger in a #comment html element but NOT trigger in a #search html element.

**Comment by amark** [comment]
Yes.

A first option is, adding a "Snooze privacy on this site for 1 week" button on the popup. Then also add to the config page a enable/disable site section.

All that needs to be done is:

Add an IF statement between these 2 lines that returns/cancels
https://github.com/eraeco/party.lol/blob/master/content.js#L95-L96
if `window.location` matches config.

Wanna help!?

**Comment by trosel** [comment]
@amark Yeah, let me play with it now that I have a hint haha. I'll reach out again if I get stuck.

**Comment by amark** [comment]
Want to join my private party? Install http://party.lol to read my secret post:
SEA{"m":{"#":"jud7q6icKsvQ2hSaIw9v~pIqWHWOhyTbSLKEfbcmanUinGXfKSG8-buPBu15YjY4._fWJsqaR8CGHMmn5fKIFNiu37WPIdi8EsZKbVi6R05M.",".":"what",":":{"ct":"6gBVDZxWakt0jYd6j7jce0uLINg/+Q4vlw7NcAVnNZce","iv":"2NnLjyK/MD/Hl9L7QUaA","s":"EvDI/VG0gfnc","v":1,"w":"/lnuTc3f2/gW+oKE.YZR/FuM3Ggp3NwYM"},">":1555021626997},"s":"Brk9Ag/yikSZjUiD7j87Xm3EN+bzp9+vDicOJF6SREIJ3+PV4inb4EbANCvpHPjYIKp8/SLa7fNVekRyAoal3A=="}

**Comment by trosel** [comment]
Two Questions:

1. What's with this line? lol https://github.com/trosel/party.lol/blob/master/content.js#L6

2. What if instead of a config page, we changed the extension to add *two* right-click options. So you could right-click in ANY input box and select "party" or "secret agent". Give user more immediate control.

**Comment by amark** [comment]
Want to join my private party? Install http://party.lol to read my secret post:
SEA{"m":{"#":"jueiagu31GoUGKrTUdui~pIqWHWOhyTbSLKEfbcmanUinGXfKSG8-buPBu15YjY4._fWJsqaR8CGHMmn5fKIFNiu37WPIdi8EsZKbVi6R05M.",".":"what",":":{"ct":"wmtoUVnHjBqMsTejdl/rwBEmNuFLBfAuM4qpQQeSMQPD7mQ5PzX35+ndNdFRg2IS1hTH2tEjj3BFGDJ1LNsLXxr34unriF7Ra+gN7oNJYSpEHqtmC3joSPiOdnvA/V2RAjz0ZDkpn7M4sNxCxoc4+ExIcaSWX/jnIcgKyuqAbHz80vNvA4+HpFPxlM2jLPIZ2pStmi8EbL+ksQ0TJimQ7hK19i3mNMDXCrEtgaMvgDRcsSfGa/dkmO16oFyyYXUpt1N4zhI2ShKsZTWwat13eauql11QwzJSctfibb7KnAORyHm1Bgpc28v2a7dep3Of5T5rDg2jvaAWeEi7bUClMatq1HYouu79JIAX4PIj7GCbRDRETCqzJTsCmOird3LoEKmtQFJAv+LnZp+uu3VZMz1GAfT84gxuKYjDdL6oOG5W7lhtjI4uYepbeo/9gZitB5XZW1416kGgtzG+Had9HVXo+IBj47kJ3Nt7vYZPgvXdNkgq1pjM/4UoEo68HM2KNPd992d670sPohr/Omeb3G2t7mEnSj/tDla+tQqiMP4GDXXb7TFTyJdEw2ImRRVPVgNrOiVjRLPyDjJpX9rwEHxDO/Om4dnhfp25Nbtj0WpCNMZyAZpky41aWKkqZSngqhd0cuyv01TXaftVIOn3EgjGYPi+b0c8ZyAVH8TZiCpJdfR2NgoihTDHL+zNpsw8IxgicYJC/fdN49TUEMXlCPmTwPICHoEKx27C3OQfK5yDHFZIRZGXccAgnrWJ9f2feamR18NW00j8ebLzdNxKTYMKHUk1wkG2K5lTpOH6p58h43DUW+ycYszJpboE52OnVrjjDWGuMO3CE7Z9dEEOE+Wfwe0MWqN4iluXEhxlWnwwsv0YtctUvopPrL0t2ODtFLaOuvV0QfjW81mRp47fyNoEbw9/1rUaECUoy7CluPM4EEiPPEou48XElYpF9NmALQLgdS52qoBUrp91UXmUsv0qWr9w2Wq7p2VkfKtLG0cD7W5QR/sRJ9KJg64kicINYVukuEsx6B1r5QNOmc01wEZWEjoWtWKojzqL6EYu75BayHyjt5wAHYYnlhSLrTPhSePBFiVkuLEJs2CWiyZp/mf0rc1M74O9vOdYDVQmdvfDg9h3bfYVVwNdcvBgdUPw1z8j+KHezzF5sCId2/Iq0FP0UvqDlmSxR4QPUNg=","iv":"an3hx0CBgPvJrASRvX6v","s":"F4bF1NTqkiUZ","v":1,"w":"u01zDAhAj0BMIRxd.YNPC6nvaGHvGEawA"},">":1555099835837},"s":"dBPsJ5D2Z3NaONywJM3yiN1KmxCwUH9v0/tJ+ksSgWIOjJjO2O38bI9LINjzuO4e4J4agWUGTicJtlfTMtYtFg=="}
